scripts/prepare/bug_reporter.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
I think it's a lot better if these prepare shell scripts live in the same directory as the corresponding agent, don't you think?

Reply:
I went back and forth on this, but I ultimately moved them because I think they're part of the repo's CI setup rather than part of the agent. They have some value as quasi-documentation w.r.t. the required secrets but making sure that the agent directory is repo-agnostic seemed preferable (and I think comments/`README.md` are more direct forms of documentation).